harvard-cape-peninsula-university-of-technology.csl

```diff
@@ -109,8 +109,8 @@
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group>
```

Comment:
If you're not getting a comma, then you're not trying the code as suggested by @rmzelle above. I just tested on csl.mendeley.com and it works as it should. Not sure how you're testing this -- one thing is that CSL (like all XML) is picky about details. Note it's ```delimiter``` not "deliminator". 
(The reason delimiter is better, btw., is that it's more robust. Should, e.g. a reference manager offer an option to just display an author and not the year, your code would produce odd results).